File: UBB/Fourth Semester/Projecting and Programming Methods/Laboratory 03/Lab3JAVA/AthleticsCompetition/src/domain/Round.java
package domain;

public class Round {
    Integer id;
    String name;

    /**
     * constructor for round
     * @param name - string, represents the name of the round
     */
    public Round(String name) {
        this.name = name;
    }

    /**
     * getter methode for id
     * @return - integer, represents the round's id
     */
    public Integer getId() {
        return id;
    }

    /***
     * setter methode for id
     * @param id - integer, represents the round's id
     */
    public void setId(Integer id) {
        this.id = id;
    }

    /***
     * getter methode for round's name
     * @return - string, represents the round's name
     */
    public String getName() {
        return name;
    }

    /**
     * setter methode for round's name
     * @param name - string, represents the round's name
     */
    public void setName(String name) {
        this.name = name;
    }
}

File: UBB/Fourth Semester/Projecting and Programming Methods/Laboratory 03/Lab3C#/AthleticsCompetition/domain/Round.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AthleticsCompetition.domain
{
    class Round
    {
        /// <summary>
        /// constructor for round
        /// </summary>
        /// <param name="id"> integer </param>
        /// <param name="name" > string </param>
        public Round(int id, string name)
        {
            this.Id = id;
            this.Name = name;
        }

        /// <summary>
        /// getter and setter for round id
        /// </summary>
        protected int Id { get; set; }

        /// <summary>
        /// getter and setter for round id
        /// </summary>
        protected string Name { get; set; }

    }
}
